package.json

```diff
@@ -16,7 +16,9 @@
     "clean": "rimraf ./dist",
     "build": "npm run clean && tsc",
     "watch": "tsc --watch",
-    "prepublishOnly": "npm run build"
+    "prepublishOnly": "npm run build",
+    "test": "jest -- ./src/__tests__/auth.spec.ts",
```

Comment:
Change this one and `npm run test` should run all the test cases @anuragpandey1115